src/express/src/routes/root.ts

```diff
@@ -40,7 +37,8 @@ export function createRootRouter(rootService: RootService) {
   // get warp routes
   router.get(
     '/warp-routes',
-    validateRequest({ query: warpRoutesQuerySchema }),
+    validateQueryParam('symbol', ZSymbolOptional),
```

Comment:
since this is being passed into getWarpRoutes, shouldn't this match `WarpRouteFilterParams`? 

in otherwords, it doesnt quite match and can we make `WarpRouteFilterParams` a schema and just use that?